lib/bigtable/cache/redisClient.js

```diff
@@ -64,14 +68,33 @@ class RedisClient {
 
     this.redis.on("end", () => debug("Redis connection has ended."));
   }
+
+  async setLastAccess(keyOrKeys) {
+
+    if (this.errorEconnrefused) {
+      throw this.errorEconnrefused
+    }
+
+    const timestamp = Date.now();
+
+    if (!Array.isArray(keyOrKeys)) {
+      keyOrKeys = keyOrKeys + "";
+      return this.redis.zadd(`${LASTACCESS_SET}`, timestamp, keyOrKeys);
+    }
+
+    return Promise.all(
+      keyOrKeys
+        .map((key) => this.redis.zadd(`${LASTACCESS_SET}`, timestamp, key))
```

Comment:
zadd should support multiple scores and values with a single redis call, currently you are making multiple calls.